src/main/java/hudson/plugins/sloccount/util/FileFinder.java

```diff
@@ -9,6 +9,7 @@
 import org.apache.tools.ant.BuildException;
 import org.apache.tools.ant.Project;
 import org.apache.tools.ant.types.FileSet;
+import org.jenkinsci.remoting.RoleChecker;
```

Comment:
After deeper investigation I believe that current implementation is fine.
FileFinder is used only by SloccountParser which already implements MasterToSlaveFileCallable.
I think that this should be enough (but I may be wrong).